### PR TITLE
ADD: Display error message returned from github if available

### LINKED ISSUE #18: Handle user not found situation.
Currently, if there is some problem with request, we will ask user to open console log. However with common cases like user not found, we should alert user about that.

https://github.com/ducthienbui97/pullrequests/blob/65f5e3a8e2cbc357c3ec99d8d49ce95164cfcad8/src/components/User.js#L20
3vi1wwzc4m

### PR DESCRIPTION
**Pull request by martskins**
fixed #18

### CODE REVIEW

File: src/components/User/index.js [hunk 1]
```diff
@@ -9,18 +9,33 @@ export default class User extends Component {
...
   };
+
+  handleError(err) {
+    let errorMessage = "There is something wrong, open log for more information!";
+    if (err.response &&
+        err.response.data &&
+        err.response.data.errors &&
+        err.response.data.errors.length > 0
+    ) {
+      errorMessage = err.response.data.errors.map(e => ("- " + e.message)).join("\r\n");
```
Comment (ducthienbui97) [inline src/components/User/index.js]: Codacy is complaining about this line

Comment (martskins) [reply src/components/User/index.js]: My bad, misread the error before. That should do it

File: src/components/User/index.js [hunk 2]
```diff
@@ -9,18 +9,33 @@ export default class User extends Component {
...
   };
+
+  handleError(err) {
+    let errorMessage = "There is something wrong, open log for more information!";
+    if (err.response &&
+        err.response.data &&
+        err.response.data.errors &&
+        err.response.data.errors.length > 0
+    ) {
+      errorMessage = err.response.data.errors.map((e) => "- " + e.message).join("\r\n");
```
Comment (ducthienbui97) [inline src/components/User/index.js]: if there's only 1 error, I don't think we should add "- " at the begining

**Comment by ducthienbui97** [comment]
Thanks for contributing.

**Pull request merged**